### ISSUE TITLE
xsstrike.tk blocked by Sophos

### ISSUE THREAD

**Issue by BrettGregson** [s0md3v/XSStrike#102]
Just a heads up, the http://xsstrike.tk/ site is blocked by Sophos with the following message:

![screenshot 2018-11-12 at 12 04 11](https://user-images.githubusercontent.com/4068025/48340398-33d49f00-e673-11e8-958f-6398da25b9e5.png)

**Comment by s0md3v** [comment]
Oh wow, I will fix this asap, thanks.

**Comment by s0md3v** [comment]
I can't really buy a domain for setting up the project homepage so I took a free one, `xsstrike.tk` but I wanted to keep the source code open so I hosted it on it github.
Well, the domain provider doesn't let users setup `CNAME` or `A` records (only nameservers) so I had to embed the github site in a fullscreen iframe as a workaround.
I hope now it makes sense why it was it was marked as malicious.

Anyway, I changed `xsstrike.tk` to `somdev.me/xsstrike` so we are fine now.
